wsd/FileServer.cpp

```diff
@@ -940,29 +942,27 @@ void FileServerRequestHandler::preprocessFile(const HTTPRequest& request,
     std::string socketProxy = "false";
     if (requestDetails.isProxy())
         socketProxy = "true";
-    Poco::replaceInPlace(preprocess, std::string("%SOCKET_PROXY%"), socketProxy);
+    replaceVar["SOCKET_PROXY"] = socketProxy;
```

Comment:
Shouldn't we include the `%` too? Why do we drop the percentage marks from all the variables?